crates/sp-domains-fraud-proof/src/fraud_proof.rs

```diff
@@ -550,6 +551,80 @@ where
     }
 }
 
+impl<Number, Hash, DomainHeader: HeaderT> fmt::Display for FraudProof<Number, Hash, DomainHeader> {
```

Comment:
Why `Display` and not `Debug`? I don't think there is a natural way to print a fraud proof, so this looks odd.

Reply:
I usually derive `Debug` and impl for `Display`. Is the format `InvalidStateTransition({domain_id:?}#{bad_receipt_hash:?}) fraud proof` look odd? how about `InvalidStateTransitionFraudProof({domain_id:?}#{bad_receipt_hash:?})`

Reply:
I guess my concern is that this is not what `Display` should do, this is not a block number or block hash that have a natural way to be displayed. If this is different from `Debug` implementation then maybe have a different custom method that returns a string.

Reply:
Okay, will change it to`Debug`